content/bare/eu/usegalaxy/notices.md

```diff
@@ -12,15 +15,3 @@ For more information, please see our [blog post](https://galaxyproject.org/news/
 Today, 21st of April 2023, a statement in support of peaceful, but resolved protest against insufficient climate politics got released and has been signed by more than 1000 scientists from German-speaking countries in Europe. Read [why members of the Freiburg, Germany branch of Galaxy Europe endorse this initiative](https://galaxyproject.org/news/2023-04-21-act-dont-criminalize/).
 
 </div>
-
-<div class="alert" style="background: #FFD500;">
-
-#### **Peace to Ukraine!**
-
-🇺🇦 The **[list of laboratories][ua-table]** that can host Ukrainian scientists can be found here. Galaxy Project has a number of positions at its EU and US sites. Contact us at **[ukraine@galaxyproject.org][ua-mail]** | Список **[лабораторій][ua-table]** які можуть прийняти українських науковців усіх рівнів доступний тут. Galaxy Project має відкриті вакансії у своīх європейських та американських осередках. Пишіть нам на **[ukraine@galaxyproject.org][ua-mail]** | **[Cписок лабораторий][ua-table]**, которые могут принять украинских ученых доступен здесь. Galaxy Project имеет открытые позиции на своих европейских и американских сайтах. Контактируйте нас используя **[ukraine@galaxyproject.org][ua-mail]** 🇺🇦
-
-</div>
-
-[ua-table]: https://bit.ly/ua-table
-[ua-form]: https://bit.ly/ua-form
-[ua-mail]: mailto:ukraine@galaxyproject.org?subject="Galaxy-UA"
```

Comment:
There's probably a way to just comment the block instead of removing it, but I'm not sure what would work exactly.

So general question: is there a good way to preview these announcements?

Reply:
You need to build it locally.

Reply:
Sure, but if I `make build` the hub pages, then where are the announcements? Or, how do I build the .eu homepage?

Reply:
here: https://galaxyproject.org/bare/eu/usegalaxy/main/?nonce=2805202#act-dont-criminalize